planning/behavior_velocity_planner/src/scene_module/blind_spot/debug.cpp

```diff
@@ -175,42 +91,44 @@ visualization_msgs::msg::MarkerArray BlindSpotModule::createDebugMarkerArray()
   visualization_msgs::msg::MarkerArray debug_marker_array;
 
   const auto state = state_machine_.getState();
-  const auto current_time = this->clock_->now();
+  const auto now = this->clock_->now();
 
   appendMarkerArray(
-    createPathMarkerArray(debug_data_.path_raw, "path_raw", lane_id_, 0.0, 1.0, 1.0),
-    &debug_marker_array, current_time);
+    debug::createPathMarkerArray(
+      debug_data_.path_raw, "path_raw", lane_id_, now, 0.6, 0.3, 0.3, 0.0, 1.0, 1.0),
+    &debug_marker_array, now);
 
   appendMarkerArray(
     createPoseMarkerArray(
-      debug_data_.stop_point_pose, state, "stop_point_pose", lane_id_, 1.0, 0.0, 0.0),
-    &debug_marker_array, current_time);
+      debug_data_.stop_point_pose, state, "stop_point_pose", module_id_, 1.0, 0.0, 0.0),
+    &debug_marker_array, now);
 
   appendMarkerArray(
     createPoseMarkerArray(
-      debug_data_.judge_point_pose, state, "judge_point_pose", lane_id_, 1.0, 1.0, 0.5),
-    &debug_marker_array, current_time);
+      debug_data_.judge_point_pose, state, "judge_point_pose", module_id_, 1.0, 1.0, 0.5),
+    &debug_marker_array, now);
 
   appendMarkerArray(
-    createPolygonMarkerArray(
-      debug_data_.conflict_area_for_blind_spot, "conflict_area_for_blind_spot", lane_id_, 0.0, 0.5,
-      0.5),
-    &debug_marker_array, current_time);
+    debug::createPolygonMarkerArray(
+      debug_data_.conflict_area_for_blind_spot, "conflict_area_for_blind_spot", module_id_, now,
+      0.3, 0.0, 0.0, 0.0, 0.5, 0.5),
+    &debug_marker_array, now);
 
   appendMarkerArray(
-    createPolygonMarkerArray(
-      debug_data_.detection_area_for_blind_spot, "detection_area_for_blind_spot", lane_id_, 0.0,
-      0.5, 0.5),
-    &debug_marker_array, current_time);
+    debug::createPolygonMarkerArray(
+      debug_data_.detection_area_for_blind_spot, "detection_area_for_blind_spot", module_id_, now,
+      0.3, 0.0, 0.0, 0.0, 0.5, 0.5),
+    &debug_marker_array, now);
 
   appendMarkerArray(
-    createObjectsMarkerArray(
-      debug_data_.conflicting_targets, "conflicting_targets", lane_id_, 0.99, 0.4, 0.0),
-    &debug_marker_array, current_time);
+    debug::createObjectsMarkerArray(
+      debug_data_.conflicting_targets, "conflicting_targets", module_id_, now, 0.99, 0.4, 0.0),
+    &debug_marker_array, now);
 
   appendMarkerArray(
-    createPathMarkerArray(debug_data_.spline_path, "spline", lane_id_, 0.5, 0.5, 0.5),
-    &debug_marker_array, current_time);
+    debug::createPathMarkerArray(
+      debug_data_.spline_path, "spline", lane_id_, now, 0.3, 0.0, 0.0, 0.5, 0.5, 0.5),
```

Comment:
@takayuki5168 
sorry one more fix is necessary then this is LGTM
```suggestion
      debug_data_.spline_path, "spline", lane_id_, now, 0.3, 0.1, 0.1, 0.5, 0.5, 0.5),
```
![image](https://user-images.githubusercontent.com/65527974/189263787-a1987134-18b1-4878-809e-073b2fd86699.png)

Reply:
Thank you!!!